### ISSUE TITLE
Issue with pip install

### ISSUE THREAD

**Issue by rdeykin** [ccsb-scripps/AutoDock-Vina#185]
I am trying to install vina using a conda environment. However, when I call

pip install vina

I get this output: 

Collecting vina
  Using cached vina-1.2.3.tar.gz (95 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [4 lines of output]
      fatal: Not a git repository (or any parent up to mount point /local)
      Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
      Version found 1.2.3 (from __init__.py)
      error in vina setup command: 'python_requires' must be a string containing valid version specifiers; Invalid specifier: '>=3.5.*'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

I am not sure what the issue could be. Any help would be greatly appreciated!

**Comment by tonyliang19** [comment]
OS: Linux Ubuntu 22.04.2

With `python==3.9.7`, `pip install vina` worked for me. If you are using conda, you could create it by the following:

```bash
conda create -y -n vina python=3.9.7
conda install -y -c conda-forge numpy swig boost-cpp sphinx sphinx_rtd_theme
pip install vina
```

**Comment by diogomart** [comment]
Fixed in #167
